### DISCUSSION TITLE
After the agent is added, the startup will output a lot of logs. How can I close these logs?

### DISCUSSION THREAD

**Discussion by CrazyLionCat** [apache/skywalking#10664] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

![image](https://user-images.githubusercontent.com/31852897/230883013-c5bb4d78-ddb6-4fa1-a995-9d31ebf439f6.png)
After the agent is added, the startup will output a lot of logs.
How can I close these logs?

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
Change log level from info to error.
